host/sgx/loadelf.c

```diff
@@ -1025,6 +1029,37 @@ static oe_result_t _patch(oe_enclave_image_t* image, size_t enclave_size)
     return result;
 }
 
+static oe_result_t _get_debug_modules(
+    oe_enclave_image_t* image,
+    oe_enclave_t* enclave,
+    oe_debug_module_t** modules)
+{
+    oe_result_t result = OE_UNEXPECTED;
+    *modules = NULL;
+    if (image->submodule)
+    {
+        oe_debug_module_t* debug_module =
```

Comment:
Should we free the `debug_module` (and `debug_module->path`) in case of failure?

Reply:
Good point. Fixed.